nolongerevil/run.sh

```diff
@@ -86,66 +85,25 @@ else
     exit 1
 fi
 
-# Set environment variables for Node.js
+# Set environment variables for Python server
 export API_ORIGIN
 export PROXY_PORT
 export CONTROL_PORT
-export INGRESS_PORT
 export ENTRY_KEY_TTL_SECONDS
 export DEBUG_LOGGING
 export DEBUG_LOGS_DIR=/data/debug-logs
-export SQLITE3_ENABLED=true
 export SQLITE3_DB_PATH=/data/database.sqlite
 
 bashio::log.info "Configuration:"
 bashio::log.info "  API_ORIGIN: ${API_ORIGIN}"
-bashio::log.info "  PROXY_PORT: ${PROXY_PORT} (container listen port)"
-bashio::log.info "  CONTROL_PORT: ${CONTROL_PORT}"
-bashio::log.info "  INGRESS_PORT: ${INGRESS_PORT}"
+bashio::log.info "  PROXY_PORT: ${PROXY_PORT} (device API)"
+bashio::log.info "  CONTROL_PORT: ${CONTROL_PORT} (control API + web UI)"
 bashio::log.info "  DEBUG_LOGGING: ${DEBUG_LOGGING}"
 bashio::log.info "  MQTT_HOST: ${MQTT_HOST}"
 bashio::log.info "  MQTT_PORT: ${MQTT_PORT}"
 bashio::log.info ""
 bashio::log.info "Nest devices will connect to: ${API_ORIGIN}"
 
-# Start the vendor API server FIRST (creates database tables)
-bashio::log.info "Starting API server..."
-cd /server || exit 1
-node dist/index.js &
-SERVER_PID=$!
-bashio::log.info "API server started (PID: ${SERVER_PID})"
-
-# Wait for server to initialize database
-bashio::log.info "Waiting for database initialization..."
-MAX_WAIT=30
-WAIT_COUNT=0
-while [ $WAIT_COUNT -lt $MAX_WAIT ]; do
-  if [ -f /data/database.sqlite ]; then
-    bashio::log.info "Database ready"
-    break
-  fi
-  sleep 1
-  WAIT_COUNT=$((WAIT_COUNT + 1))
-done
-
-if [ $WAIT_COUNT -eq $MAX_WAIT ]; then
-  bashio::log.fatal "Database initialization timeout after ${MAX_WAIT} seconds"
-  kill $SERVER_PID 2>/dev/null
-  exit 1
-fi
-
-# Start the frontend (Ingress UI + MQTT initialization)
-bashio::log.info "Starting frontend web UI..."
-cd /frontend || exit 1
-node dist/index.js &
-FRONTEND_PID=$!
-bashio::log.info "Frontend started (PID: ${FRONTEND_PID})"
-
-# Wait for any process to exit (both should stay running)
-wait -n $FRONTEND_PID $SERVER_PID
-EXIT_CODE=$?
-
-# If one exits, kill the other
-bashio::log.warning "A process exited with code ${EXIT_CODE}, shutting down..."
-kill $FRONTEND_PID $SERVER_PID 2>/dev/null
-exit $EXIT_CODE
+# Start the Python server (handles everything: API, control, and web UI)
+bashio::log.info "Starting server..."
+exec python3 -m nolongerevil.main
```

Comment:
We should add a chore to host this with a real ASGI server that's multithreaded., not urgent now.